apps/api/src/domain/models/context/commands/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './connect-resources-with-note';
 export * from './create-note-about-resource';
+export * from './translate-note-about-resource';
```

Comment:
Can you rename all files consistently with `translate-note`

Reply:
done